.prettierrc.json

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Do we need this empty config file?

Reply:
If the documentation is to be believed, yes.

Reply:
Nope, removing.